config/.rubocop.yml

```diff
@@ -0,0 +1,9 @@
+Style/StringLiterals:
+  EnforcedStyle: double_quotes
```

Comment:
If the rule is set to double quotes why do we have single quotes changes?

Reply:
There was some discrepancy is docs. The folder structure was incorrect. I have updated it